app/views/layouts/application.html.erb

```diff
@@ -116,6 +116,14 @@
               </li>
             </ul>
           </div>
+          <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+            <h2 class="govuk-heading-m">Give feedback</h2>
+            <div class="govuk-footer__meta-custom">
+              <ul class="govuk-footer__meta-custom govuk-list govuk-!-font-size-16">
+                <li><%= govuk_link_to "Give feedback to help us improve Register trainee teachers", Settings.feedback_link_url %></li>
+              </ul>
+            </div>
+          </div>
```

Comment:
Issue here is that footer doesnt align neatly; could do with some advise on how to make it line up correctly. 
<img width="1792" alt="Screenshot 2022-05-17 at 13 13 18" src="https://user-images.githubusercontent.com/44261976/168808188-6d96e6f7-e72f-4bae-bf01-c0fbaafbb72b.png">

Reply:
Check how it's done in the prototype. There is another width container and then a grid row with 50% columns for support and feedback. @edwardhorsford also has a custom meta class in there and may be able to advise if you're still stuck.

Reply:
This section needs grid columns to make it work. @richardpattinson we can pair on this if you like.

Reply:
I think ive got ti to work by copying element from the prototype. Give it a look and it should be good